### ISSUE TITLE
tunnelto.dev subdomains not working

### ISSUE THREAD

**Issue by stealthybox** [agrinman/wormhole#1]
Hi, cool project!
I was playing with it and instantly noticed the subdomain flag.

Are subdomains supposed to work on the public instance?
Do I need to configure an API key for that to work?

I expected this command to give me back a public URL similar to `stealthybox.tunnelto.dev` or `stealthybox-aexsh3w5.tuneelto.dev`, but instead, the `-s` option seems ignored.
```
wormhole start -s stealthybox -p 31888
Welcome to wormhole!

<trunctated>

Wormhole activated on: https://aexsh3w5.tunnelto.dev
```
Did I misunderstand that this is supposed to only be an overriden host-header that's passed into the proxied request?

If not and I'm understanding this feature correctly, we can probably implement an error or warning message and include some docs around API keys or the capabilities of private instances.

**Comment by agrinman** [comment]
Hi @stealthybox, this is currently by design (though you're right that a helpful error message is needed). For the publicly hosted version, we're currently only allowing random subdomains. If you deploy it yourself, this is controlled by enabling a secret key as noted in the readme.

I like the idea of adding random characters to a specified subdomain though -- we can definitely support that in the publicly hosted deployment!

**Comment by agrinman** [comment]
Fixed in #eb4fb89

**Comment by stealthybox** [comment]
nice! thanks!
